test/Swap.test.ts

```diff
@@ -36,22 +36,28 @@ describe("Swap RBTC", function () {
     const senderBalance: BigNumber = await sideTokenBtc.balanceOf(sender.address);
 
     expect(senderBalance.toString()).to.equals(oneEther.toString());
-    await sideTokenBtc.connect(sender).approve(swapRBTC.address, halfEther);
 
     await deployer.sendTransaction({
       to: swapRBTC.address,
       value: oneEther
     });
+    
+    await sideTokenBtc.connect(sender).approve(swapRBTC.address, halfEther);
 
-    const balanceBeforeSwap = await sender.getBalance();
-    const receipt = await swapRBTC.connect(sender).swapWRBTCtoRBTC(halfEther, sideTokenBtc.address);
-    const balanceAfterSwap = await sender.getBalance();
+    const balanceBeforeSwap = await sender.getBalance(); // 100
+    const response = await swapRBTC.connect(sender).swapWRBTCtoRBTC(halfEther, sideTokenBtc.address); // 50
+    const receipt = await response.wait();
+    console.log(response);
```

Comment:
I think that would be a good if we remove the tests console.log before do the PR.

Reply:
yeah don't worry cuz that whole test needs a variable renaming and restyling. The PR is not intended to be definitive, so that's why my comment in the pr.